.config/webpack/base.js

```diff
@@ -40,15 +39,18 @@ module.exports = (options = {}) => {
             rules: [
                 {
                     test: /demo[\\\/]index.html?$/,
-                    loader: 'file-loader?name=index.[ext]'
+                    loader: 'file-loader',
+                    options: {
+                        name: 'index.[ext]'
+                    }
                 },
                 {
                     test: /\.csv$/,
                     loader: 'raw-loader'
                 },
                 {
                     test: /\.ts(x?)$/,
-                    include: /node_modules[\\\/](highlight[.]js)[\\\/]/,
+                    include: /node_modules[\\\/](highlight[.]js|d3-format)[\\\/]/,
```

Comment:
`d3-format` v2 is now bundled in ES6, we need to transpile it to ES5 for our IE11 support.